association-frontend/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+
+{
+    "compilerOptions": {
+        "target": "es5",
+        "lib": ["dom", "dom.iterable", "esnext"],
+        "allowJs": true,
+        "skipLibCheck": true,
+        "strict": false,
+        "forceConsistentCasingInFileNames": true,
+        "noEmit": true,
+        "esModuleInterop": true,
+        "module": "esnext",
+        "moduleResolution": "node",
+        "resolveJsonModule": true,
+        "isolatedModules": true,
+        "jsx": "preserve",
+        "baseUrl": ".",
+        "paths": {
+            "~/components/*": ["./components/*"],
+            "~/helpers/*": ["./helpers/*"]
+        }
```

Comment:
right now all of the code is inside the root of "association-frontend", we should consider moving it into `src/`